recipe/build.sh

```diff
@@ -13,6 +13,9 @@ unset SUBDIR
         --extra-libs="`pkg-config --libs zlib`" \
         --enable-pic \
         --enable-gpl \
+        --enable-version3 \
+        --enable-hardcoded-tables \
```

Comment:
Also this does what you expect and adds some hardcoded tables to the libraries instead of runtime computing them. The tradeoffs per usual are time vs. memory usage and binary size. Any reference I have seen seems to suggest these tables are small (e.g. KBs). The most recent FFmpeg mailing list item on this was from a decade ago. So I think this option has just become non-controversial since for rather obvious reasons.

That said, there is this [suggestion]( https://trac.ffmpeg.org/wiki/CompilationGuide ) that it may not make for much of an improvement, but doesn't result in an increase in `libavcodec` size by 15%.

FWIW I have no strong feelings on this either way. Just sharing the info.